cfgov/tccp/templates/tccp/widgets/checkbox_select.html

```diff
@@ -0,0 +1,5 @@
+{% with id=widget.attrs.id %}<div{% if id %} id="{{ id }}"{% endif %}{% if widget.attrs.class %} class="{{ widget.attrs.class }}"{% endif %}>{% for group, options, index in widget.optgroups %}{% if group %}
+  <div><label>{{ group }}</label>{% endif %}{% for option in options %}
+    {% include option.template_name with widget=option %}{% endfor %}{% if group %}
+  </div>{% endif %}{% endfor %}
+</div>{% endwith %}
```

Comment:
Should this be formatted with indents/newlines for greater readability?

Reply:
This is based on Django's template [here](https://github.com/django/django/blob/61a986f53d805e4d359ab61af60a2dcd55befe25/django/forms/jinja2/django/forms/widgets/multiple_input.html), with the only minor change being stripping out the wrapping `<div></div>`. I tried to keep the file the same otherwise.

Reply:
Does Django condense it down for some reason? (Performance over the wire?)

Reply:
I don't know, maybe to reduce unnecessary whitespace in rendered templates? I'm happy to change if you prefer.

Reply:
It makes sense to copy what they have, but maybe add a code comment at the top linking to the copied template, so it's clearer why it's the way it is. (If you feel like it, perhaps an issue would be opened with django regarding the formatting)

Reply:
Added clarifying comment in a0a60e0.